.github/workflows/test-native-gradle-plugin.yml

```
@@ -65,9 +80,9 @@ jobs:
           java-version: ${{ matrix.java-version }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
       - name: "❓ Check and test the plugin"
-        run: ./gradlew :native-gradle-plugin:functionalTest -DgradleVersion=${{ matrix.gradle-version }}
+        run: ./gradlew :native-gradle-plugin:functionalTest -DgradleVersion=${{ matrix.gradle-version }} --tests ${{ matrix.test }}
       - name: "❓ Check and test the plugin with configuration cache"
-        run: ./gradlew :native-gradle-plugin:configCacheFunctionalTest -DgradleVersion=${{ matrix.gradle-config-cache-version }}
+        run: ./gradlew :native-gradle-plugin:configCacheFunctionalTest -DgradleVersion=${{ matrix.gradle-config-cache-version }}  --tests ${{ matrix.test }}
       - name: "📜 Upload functional tests results"
         if: always()
         uses: actions/upload-artifact@v2
```

Please bump all uses of `actions/upload-artifact` to `v3` the next time you push:
```suggestion
        uses: actions/upload-artifact@v3
```